### ISSUE TITLE
Tags can't be generated on a different path than the root

### ISSUE THREAD

**Issue by kriansa** [eapache/starscope#163]
Because the tag generation and exporting always use relative paths, you can't just use a different output path using `file` option, because if it's inside a folder, those tags will be parsed as if they were relative to the project's folder, not to the output folder we specified.

A possible fix for that would be to use absolute paths for generating or exporting tags.

#### Example:
```shell
starscope \
  --exclude vendor/bundle \
  --exclude node_modules \
  --exclude public \
  --exclude .git \
  --file ".meta/.starscope.db" \
  --export ctags,".meta/tags"
```

**Comment by eapache** [comment]
Does this bug occur when using a different database folder (with `--file`) or when using a different output folder (with `--export ctags,FOO`), or both? It's not entirely clear from your description.

**Comment by kriansa** [comment]
I haven't tested using different paths for database and exported ctags, however, if you ungzip the database, you'll see that every file there has a relative path according to the working dir when generating the tags, not to db destination, and _maybe_ that can be the an issue as well.

**Comment by eapache** [comment]
Fix released in v1.5.4.
